src/com/github/arseniydubrovski/dev5/MaximalProductivity.java

```diff
@@ -0,0 +1,61 @@
+package github.arseniydubrovski.dev5;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * calculate optimal composition of team you can find information about this method (Dynamic
+ * programming in-advance algorithm) here  https://en.wikipedia.org/wiki/Knapsack_problem
+ */
+public class MaximalProductivity {
+
+  private Integer amountOfMoney;
+
+  /**
+   * Constructor
+   *
+   * @param amountOfMoney sum of money at which we calculate optimal composition of team
+   */
+
+  public MaximalProductivity(Integer amountOfMoney) {
+    this.amountOfMoney = amountOfMoney;
+  }
+
+  /**
+   * calculate List optimal qualifications of team
+   * calculate maxProductivity
+   *
+   * @return List optimal qualifications of team
+   */
+  public List<String> calculator() {
```

Comment:
Looks similar to the MinimalPrice - bad design as you re-use almost the same logic